shell/src/test/java/tachyon/shell/TfsShellTest.java

```diff
@@ -295,16 +295,14 @@ public void copyFromLocalOverwriteTest() throws Exception {
     String[] cmd1 = {"copyFromLocal", testFile1.getPath(), tachyonFilePath.getPath()};
     mFsShell.run(cmd1);
     Assert.assertEquals(getCommandOutput(cmd1), mOutput.toString());
+    mOutput.reset();
```

Comment:
Similarly, the tests can be split into individual ones per command. This will make the commands more modularized. But this can be done in a future PR if you feel this PR is too big.